src/components/Reports/Reports.js

```diff
@@ -54,10 +55,14 @@ class Reports extends React.Component {
     this.setState({ domainSelected: domainFound });
   };
 
+  addDays = (date, days) => {
+    return date.setDate(date.getDate() + days);
```

Comment:
Please, take care of this, you are mutating the input object, it is unexpected and [astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment).